test/test_broker/mocks/agent.js

```diff
@@ -70,6 +73,19 @@ function preUpdate(expectedReturnStatusCode) {
     .reply(expectedReturnStatusCode || 200, {});
 }
 
+function getPostCreateProcessingState(body, status) {
+  return nock(agentUrl)
+    .replyContentLength()
+    .get('/v1/processing/postcreate')
```

Comment:
Can we name the feature as lifecycle.post[create|update] ? We already have `lifecycle.preupdate`. In that case, it will be consistent.

Reply:
The difference in that cases is that this is an async polling, for the service instance to finish with some internal logic after the create/update. The preupdate is a syncronous action, which is executed without polling afterwards.
It may be desirable to add an asynchronous preupdate (first call POST and then poll with get GET) in the future, and then the featureflag `lifecycle.preupdate` would not support that.

Reply:
We could call the feature `lifecycle.async.postupdate` and `lifecycle.async.preupdate`, which calls the router `GET /lifecycle/postupdate`. For the preupdate case it could be `POST /lifecycle/postupdate?async=true`

Reply:
makes sense! 👍